apps/docs/.vitepress/data/composables.data.ts

```diff
@@ -11,7 +11,12 @@ export { data };
 
 export default defineLoader({
   async load(): Promise<Data> {
-    const composablesList = readdirSync("../../packages/composables/src/", {
+    const cwd = process.cwd();
+    const projectRootDir = cwd.endsWith('/apps/docs')
```

Comment:
Please check why composables list is empty. https://github.com/shopware/frontends/pull/1327#issuecomment-2392363766

Reply:
Fixed: https://developer.shopware.com/frontends/packages/composables/#composables-list